### DISCUSSION TITLE
Highlight selected precipitation bar

### DISCUSSION THREAD

**Discussion by NikunjK8** [breezy-weather/breezy-weather#1158] [Ideas]
### Describe your idea

A very minor suggestion, if possible please highlight the bar selected in precipitation graph so you can be sure you've selected the correct one. Specially it's hard to notice when you have bigger thumbs.

![IMG_20240626_091036](https://github.com/breezy-weather/breezy-weather/assets/99175083/efc95ff4-1e8e-4877-9a39-02d98d43fb69)

### Other details

There are alternative solutions like showing the oval (which contains time) directly over the selected bar instead on a fixed line. it'll be less useful than highlighted bars because two bars can have same height but still better than current implementation.
Maybe you can think of any other better idea?

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
I can see a guideline now, I think it was added in v6.0.0. Closing as implemented
